examples/plugins/date/main.go

```diff
@@ -49,5 +53,11 @@ func DateCommand() (*cobra.Command, error) {
 		},
 	}
 
+	// default validation provided by validator package
+	validationErr := validator.Validate(cmd)

```

Comment:
Yes.. while this is ok. It makes no sense. 

Validation is development feature but we are using it at runtime. This needs to change.